test/types.ts

```diff
@@ -0,0 +1,31 @@
+import { AbortController as AbortControllerShim, AbortSignal as AbortSignalShim } from "..";
+
+function signalCallback(as: AbortSignal) { }
+function signalShimCallback(as: AbortSignalShim) { }
+function controllerCallback(ac: AbortController) { }
+function controllerShimCallback(ac: AbortControllerShim) { }
+function abortCallback(this: AbortSignal, ev: Event) { }
+
+const controller = new AbortController();
+const controllerShim = new AbortControllerShim();
+const signal = controller.signal;
+const signalShim = controllerShim.signal;
+
+signalCallback(signal);
+signalCallback(signalShim);
+
+signalShimCallback(signal);
```

Comment:
We can assign `AbortSignalShim` value to `AbortSignal` variable, but we cannot do the reverse.

Reply:
I confirmed that this line was not a type error. However, I don't think we should consider this is a correct code because it causes a runtime type error. So would you delete this line from the test?